package.json

```
@@ -32,28 +32,28 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@nestjs/common": "^6.0.0",
-    "@nestjs/core": "^6.0.0",
-    "mqtt": "^3.0.0",
-    "reflect-metadata": "^0.1.12",
-    "rimraf": "^2.6.2",
-    "rxjs": "^6.3.3"
+    "@nestjs/common": "^8.2.0",
+    "@nestjs/core": "^8.2.0",
+    "mqtt": "^4.3.1",
```

`mqtt` version `4.3.2` does not work, because they use [`collections`](https://github.com/montagejs/collections), which modify `Map`, `Set`, `Array` (and probably some other) Node methods. See mqttjs/MQTT.js#1392 and also mikro-orm/mikro-orm#2597 (esp from [this comment](https://github.com/mikro-orm/mikro-orm/discussions/2597#discussioncomment-1912286) on).

```suggestion
    "mqtt": "4.3.1",
```